### ISSUE TITLE
Word missing in sentence on download page

### ISSUE THREAD

**Issue by NekoJonez** [WordPress/wporg-main-2022#286]
https://translate.wordpress.org/projects/meta/wordpress-org/nl-be/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=16047264&filters%5Btranslation_id%5D=105620210

This currently says: "Recommend PHP [recommended_php] or greater and MySQL [recommended_mysql] or MariaDB version [recommended_mariadb] or greater."

But I think a "version" is missing... So, shouldn't it say:

"Recommend PHP [recommended_php] or greater and MySQL **version** [recommended_mysql] or MariaDB version [recommended_mariadb] or greater."

**Comment by Otto42** [comment]
No, the extra word version is not needed there. 

MySql x.xx vs MySql version x.xx is largely a matter of choice.

**Comment by pkevan** [comment]
I think we should be consistent between the wording on both hosting and requirements page:

https://wordpress.org/about/requirements/
```
PHP version 7.4 or greater.
MySQL version 5.7 or greater OR MariaDB version 10.3 or greater.
```

https://wordpress.org/download/
```
Recommend PHP 7.4 or greater and MySQL 5.7 or MariaDB version 10.3 or greater.
```

**Comment by NekoJonez** [comment]
> I think we should be consistent between the wording on both hosting and requirements page:
> 
> https://wordpress.org/about/requirements/
> 
> ```
> PHP version 7.4 or greater.
> MySQL version 5.7 or greater OR MariaDB version 10.3 or greater.
> ```
> 
> https://wordpress.org/download/
> 
> ```
> Recommend PHP 7.4 or greater and MySQL 5.7 or MariaDB version 10.3 or greater.
> ```

True, either we drop the "version" on the hosting page or add it on the download page.

**Comment by pkevan** [comment]
Fixed in #287
